google/cloud/aiplatform/constants.py

```diff
@@ -29,3 +29,20 @@
     "file-list",
 )
 BATCH_PREDICTION_OUTPUT_STORAGE_FORMATS = ("jsonl", "csv", "bigquery")
+
+MOBILE_TF_MODEL_TYPES = {
+    "MOBILE_TF_LOW_LATENCY_1",
+    "MOBILE_TF_VERSATILE_1",
+    "MOBILE_TF_HIGH_ACCURACY_1",
+}
+
+# Defined by gs://google-cloud-aiplatform/schema/trainingjob/definition/automl_image_*
+# Format: "prediction_type": set() of model_type's
+#
+# NOTE: When adding a new prediction_type's, ensure it fits the pattern
+#       "automl_image_{prediction_type}_*" used by the YAML schemas on GCS
+AUTOML_IMAGE_PREDICTION_MODEL_TYPES = {
```

Comment:
For later: use the same information in schema types: https://github.com/googleapis/python-aiplatform/blob/master/google/cloud/aiplatform/v1beta1/schema/trainingjob/definition_v1beta1/types/automl_image_classification.py#L96

Reply:
Captured in [b/177079208](http://b/177079208)